.github/workflows/build-packages.yml

```diff
@@ -17,11 +17,18 @@ jobs:
         id: go
 
       - name: Set environment
-        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
+        run: |
```

Comment:
@fako1024 : what's the recommended way to test this? I want to arrive at something like

```
gpctl - v4.0.0:
    Build time:     Aug  4 16:23:07 2023 UTC
    Git hash:       d1e834663f67741bac66b63bd566a3dd9008bffe
    Go versions:    go1.20.5
```

Reply:
Thanks! I faintly remember you creating a few throwaway tags.

Will also test the cd ../ then.